cli/compose/loader/loader.go

```diff
@@ -358,10 +363,18 @@ func resolveEnvironment(serviceConfig *types.ServiceConfig, workingDir string) e
 
 	serviceConfig.Environment = environment
```

Comment:
nit: To be consistent I think this line should be at the end of the function, right before `return nil`